src/contracts/interfaces/IClaimingManager.sol

```diff
@@ -0,0 +1,111 @@
+
+    /// @notice Delay in timestamp before a posted root can be claimed against
+    function activationDelay() external view returns (uint32);
+
+    /// @notice Mapping: account => the address of the entity that can claim payments on behalf of the accounts
+    function recipients(address account) external view returns (address);
+
+    /// @notice Mapping: recipient => token => total amount claimed
+    function cumulativeClaimed(address recipient, IERC20 token) external view returns (uint256);
+
+    /// @notice the commission for all operators across all avss
+    function globalCommissionBips() external view returns (uint16);
+
+    /// EXTERNAL FUNCTIONS ///
+
+    /**
+     * @notice Initializes the contract
+     * @param initialOwner The address of the initial owner of the contract
+     * @param _paymentUpdater The address of the entity that can update the payment merkle roots
+     * @param _activationDelay Delay in timestamp before a posted root can be claimed against
+     * @param _globalCommissionBips The commission for all operators across all avss
+     * @dev Only callable once
+     */
+    function initialize(address initialOwner, address _paymentUpdater, uint32 _activationDelay, uint16 _globalCommissionBips) external;
+
+    /**
+     * @notice Sets the address of the entity that can update the payment merkle roots
+     * @param _paymentUpdater The address of the entity that can update the payment merkle roots
+     * @dev Only callable by the contract owner
+     */
+    function setPaymentUpdater(address _paymentUpdater) external;
+
+    /**
+     * @notice Sets the delay in timestamp before a posted root can be claimed against
+     * @param _activationDelay Delay in timestamp before a posted root can be claimed against
+     * @dev Only callable by the contract owner
+     */
+    function setActivationDelay(uint32 _activationDelay) external;
+
+    /**
+     * @notice Sets the global commission for all operators across all avss
+     * @param _globalCommissionBips The commission for all operators across all avss
+     * @dev Only callable by the contract owner
+     */
+    function setGlobalCommission(uint16 _globalCommissionBips) external;
+
+    /**
+     * @notice Sets the address of the entity that can claim payments on behalf of the account
+     * @param account The account whose recipient is being set
+     * @param recipient The address of the entity that can claim payments on behalf of the account
+     * @dev Only callable by the account or their paymentRecipient
+     */
+    function setRecipient(address account, address recipient) external;
+
+    /**
+     * @notice Creates a new distribution root
+     * @param root The merkle root of the distribution
+     * @param paymentsCalculatedUntilTimestamp The timestamp until which payments have been calculated
+     * @dev Only callable by the paymentUpdater
+     */
+    function submitRoot(bytes32 root, uint32 paymentsCalculatedUntilTimestamp) external;
```

Comment:
might also want a only-owner function for cancelling roots before they activate or something?

Reply:
tbh prefer pausing here

Reply:
pausing as-in pausing of claiming?
seems like then you'd be forced to upgrade to recover, so I'm not seeing how that's really better.
we could do both?